editor/plugins/script_editor_plugin.h

```diff
@@ -325,7 +325,8 @@ class ScriptEditor : public PanelContainer {
 
 	void _close_tab(int p_idx, bool p_save = true, bool p_history_back = true);
 
-	void _close_current_tab();
+	void _close_save_current_tab() { _close_current_tab(); } // callable_mp can't use default arguments :/
```

Comment:
`callable_mp` can't, but `connect` takes parameters. So you could have used `erase_tab_confirm->connect("confirmed", callable_mp(this, &ScriptEditor::_close_current_tab), varray(true));`.

Reply:
Right...